src/Components/MainPage/MainPage.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import SearchBar from "./SearchBar/SearchBar";
+import MovieList from "./MovieList/MovieList";
+import TrendList from "./TrendList/TrendList";
+
+const MainPage = ({ APIKEY }) => {
+  const [movies, setMovies] = useState([]);
+  const [searchValue, setSearchValue] = useState("");
+
+  const handleSubmit = async (e) => {
+    if (!searchValue) {
+      return;
+    }
+
+    e.preventDefault();
+    const data = await fetch(
+      `https://api.themoviedb.org/3/search/movie?api_key=${APIKEY}&query=${searchValue}`
+    );
+    const indMovies = await data.json();
+    setMovies(indMovies.results);
+    setSearchValue("");
+  };
+
+  return (
+    <div>
+      <SearchBar
+        handleSubmit={handleSubmit}
+        searchValue={searchValue}
+        setSearchValue={setSearchValue}
```

Comment:
Usually the callbacks in props are called `on*`, like `onChange` or `onSubmit`.  I would recommend following this rule, it would make your code easier to read for others

The general convention is described here:
https://jaketrent.com/post/naming-event-handlers-react/